tests/v2v_test.go

```diff
@@ -0,0 +1,55 @@
+package tests_test
+
+import (
+	"log"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+)
+
+// template parameters
+const (
+	v2vServiceInstance    = "tests/manifests/import-vm-v2v-apb.yml"
+	dstv2vServiceInstance = "/tmp/test-v2v.json"
+)
+
+// Array of enviroment variables to get from testing env
+var listofEnvVars = []string{"ocpID", "ocpPass", "VMWareURL", "VMName", "VMWareID", "VMWarePass"}
+
+// Map to store Enviroment variables
+var v2vEnvVars map[string]string
```

Comment:
I believe you can also use struct and iteratively fetch the value like this: http://www.golangprograms.com/how-to-get-struct-variable-information-using-reflect-package.html, But I'm okay with `map` as well.

Reply:
I feel `map` is fine here as storing them in a loop while taking input from array is easy and works

Reply:
As we have discussed these config options will move to config file in future.